Frends.JSON.ConvertXMLStringToJToken/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.1] - 2023-11-28
```

Comment:
Changing the Result object is a breaking change imo, so I think that minor version should be updated.

Reply:
I've tested this and it's not breaking change.